src/requests/getSavedProperties.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+const getSavedProperties = (setProperties, setAlert) => {
+  const endpoint =
+    "http://localhost:4000/api/v1/Favourite?populate=propertyListing";
+
+  return axios
+    .get(endpoint)
+    .then((response) => {
+      setProperties(response.data);
+      if (response.data.length === 0) {
+        setAlert({ message: "No saved property!" });
```

Comment:
Currently if you dont have any saved properties and you go to the saved properties page you get an error in the console complaining about, isSuccess being undefined. 
It is because in the Alert component in the Alert.Proptypes you are setting isSuccess as a required prop, but here you only set it the message. 
If the props are set to required you have to pass set, here you would have to pass in isSuccess: false with the message prop. Currently why it is working is because undefined is a falsy value, therefor it will show the right colour (red) but for the wrong reason.

I think below would be the same but in your case you can just remove the else block entirely, or you have to pass in the isSuccess default value.